cocotb/handle.py

```diff
@@ -146,7 +146,14 @@ def __getattr__(self, name):
             return object.__getattr__(self, name)
 
 class RegionObject(SimHandleBase):
-    """Region objects don't have values, they are effectively scopes or namespaces."""
+    """A region object.
```

Comment:
Perhaps
```suggestion
    """A region object, such as a scope or namespace.
```
since the first line is often used prominently in docstrings, and as it there's nothing conveyed.

Reply:
Done.